### DISCUSSION TITLE
something about performance of the pipe operator

### DISCUSSION THREAD

**Discussion by xl-better** [vesoft-inc/nebula#5796] [Q&A]
I read the following description "The pipeline itself needs to be serialized and deserialized, which is executed in a single thread." in the "NebulaGraph Database manual", which i don't understand. `A | B` is actually a dependency of A on B, and i don't think it involves serialize and deserialize, may someone give some instructions?

**Comment by QingZ11** [comment]
As mentioned in the [linked response](https://github.com/vesoft-inc/nebula/issues/5792#issuecomment-1876877044), the time for serialization and deserialization is not involved here. The documentation has already been updated to address this. Thank you for your feedback in helping us improve the documentation.

正如链接的对应回复所说，这里是不涉及到序列化和反序列化的时间的，文档已经对此做了修复。谢谢你的反馈帮忙完善了文档 ❤️
